<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateMasks</class>
 <widget class="QMainWindow" name="GenerateMasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="imgBtn">
       <property name="text">
        <string>Image Directory</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="imgLabel">
       <property name="text">
        <string>Image Directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="maskBtn">
       <property name="text">
        <string>Mask Directory</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="maskLabel">
       <property name="text">
        <string>Mask Directory</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="modBox">
       <item>
        <property name="text">
         <string>cyto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>nuclei</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tissuenet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>livecell</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cyto2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>general</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CPx</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TN1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TN2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TN3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LC1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LC2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LC3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LC4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="diaLabel">
       <property name="text">
        <string>Diameter</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="diamSpin"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="flowLabel">
       <property name="text">
        <string>Flow</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="probLabel">
       <property name="text">
        <string>Probability</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="flowSpin">
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.040000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="probSpin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="modLabel">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="chanLabel">
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="chanLine"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="edgeCheck">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ignore cells on the edge of the image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>No Edge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="replace">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check to replace exsiting masks&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Replace Existing Masks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="countBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add count.csv to masks folder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Create Cell Count File</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="genMaskBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Masks</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
